qt_dotgraph/src/qt_dotgraph/node_item.py

```diff
@@ -75,6 +73,18 @@ def __init__(self, highlight_level, bounding_box, label, shape, color=None, pare
 
         self.hovershape = None
 
+    def parse_shape(self, shape, bounding_box):
+        if shape == 'box' or shape == 'rect' or shape == 'rectangle':
+            self._graphics_item = QGraphicsRectItem(bounding_box)
+        elif shape == 'ellipse' or shape == 'oval' or shape == 'circle':
+            self._graphics_item = QGraphicsEllipseItem(bounding_box)
+        elif shape == 'box3d':
+            self._graphics_item = QGraphicsBox3dItem(bounding_box)
+        else:
+            #raise ValueError("Invalid shape '"+shape+"'")
+            print("Invalid shape '"+shape+"', defaulting to ellipse")
```

Comment:
Also print the error message to `stderr`:

```
print('...', file=sys.stderr)
```

This will require the following at the top of the file:

```
from __future__ import print_function

import sys
```